studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.tsx

```diff
@@ -77,13 +79,19 @@ const NavigationBar: FC<Props> = ({}) => {
               : []),
             <Divider key="d1" light />,
             <Dropdown.Misc key="theme">
-              <div className="w-[200px] pt-1">
-                <Toggle
-                  size="small"
-                  label="Enable dark mode"
-                  checked={ui.isDarkTheme}
-                  onChange={() => ui.toggleTheme()}
-                />
+              <div className="w-[240px] py-1 flex items-center justify-between">
+                <Typography.Text>Theme</Typography.Text>
+                <Select
+                  value={value}
```

Comment:
value can listen to the UiStore directly, there's no need to keep a separate `value` state

i.e `<Select value={ui.themeOption} />`

Reply:
I already tried using that, but it did not update the value prop in `ThemeSettings`. I worked as expected in `NavigationBar`. What am I missing?

```tsx
function ThemeSettings() {
  const { ui } = useStore()

  return (
    <Panel
      title={[
        <Typography.Title key="panel-title" level={5}>
          Theme
        </Typography.Title>,
      ]}
    >
      <Panel.Content>
        <Select
          value={ui.themeOption}
          onChange={(e: any) => ui.onThemeOptionChange(e.target.value)}
        >
          <Select.Option value="system">System default</Select.Option>
          <Select.Option value="dark">Dark</Select.Option>
          <Select.Option value="light">Light</Select.Option>
        </Select>
      </Panel.Content>
    </Panel>
  )
}
```